packages/graphql/src/translate/translate-read.ts

```diff
@@ -44,6 +44,10 @@ function translateRead({ node, context }: { context: Context; node: Node }): [st
     const connectionStrs: string[] = [];
     const interfaceStrs: string[] = [];
 
+    if (node.queryOptions?.defaultLimit && !optionsInput.limit) {
+        optionsInput.limit = node.queryOptions.defaultLimit;
+    }
```

Comment:
**suggestion**: Two minor things that worry me about this approach
* this is duplicate from `create-projection-and-params`, I think some common functions to handle `limit` could help avoid duplication.
* Modifying an existing object (`field.args.options`)

Maybe it would be worth some function such as 

```typescript
function getLimit(node: Node, field: ResolveTree): { limit?: Integer} {
// .....
}
```

This way, we can obtain the real limit options (including default), without modifying our pseudo global object `field.args.options`